images/ansible/roles/common/tasks/redhat.yml

```diff
@@ -16,6 +16,18 @@
   yum:
     name: "{{ common_redhat_epel_rpm }}"
 
+- name: add cloud-init copr repo
+  get_url:
+    url: "{{ common_redhat_cloud_init_copr_rpm }}"
+    dest: /etc/yum.repos.d/cloud-init-copr.repo
+
+- name: add yum priority
+  lineinfile:
+    path: /etc/yum.repos.d/cloud-init-copr.repo
+    regexp: '^priority='
+    line: 'priority=1'
+  when: ansible_distribution == "Amazon"
```

Comment:
Is this required only on Amazon?

Reply:
Yes, Amazon Linux adds an optional yum plugin for "priorities", that is a horrid mess to deal with.